boards/arm/efr32_radio/efr32_radio_brd4161a.dts

```diff
@@ -68,3 +68,9 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&i2c0 {
+	location-sda = <GECKO_LOCATION(16) GECKO_PORT_C GECKO_PIN(11)>;
```

Comment:
Using `location-*` properties is deprecated. You should convert these changes so that they use the Pinctrl API: https://docs.zephyrproject.org/latest/hardware/pinctrl/index.html

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/i2c/i2c_gecko.c
They didnt use the pinctrl api.
Do you want me to implement pinctrl apis in i2c_gecko.c driver?

Reply:
That would be required for the Gecko I2C driver to use pinmuxing. You could base your work off of the Gecko UART driver: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/serial/uart_gecko.c